wikipedia-sentences/app.py

```diff
@@ -28,30 +33,44 @@ def print_topk(resp, sentence):
                 continue
             print(f'> {idx:>2d}({score:.2f}). {match.text}')
 
-
 def index(num_docs):
-    f = Flow().load_config('flows/index.yml')
+    f = Flow().add(uses=MyTransformer).add(uses=NumpyIndexer)
```

Comment:
Not sure about this, just that personally I am more comfortable with the pythonic way

Reply:
OK. Because I don't really mind WHICH we choose, but all examples should use the same format. I have a meeting on examples 2.0 with @lusloher @FionnD @nan-wang @theUnkownName so we'll work something out there.

Incidentally, other things we should keep consistent:

- command line args: `python -t app.py index`, not `python app.py index`, etc
- dir structure: Flows in `./flows`, data in `./data`, etc
- filenames: script to get data named `get_data.sh`, etc
- variable names: like `docs` and `doc`, etc as @maximilianwerk mentioned
- README format (@FionnD and @aga11313 created a great template for that)

All of this makes it easier for a user to learn a basic example then smoothly upgrade their knowledge to a more complex example

Reply:
Above comment is more as a reminder to myself of what to discuss in that meeting 😃

Reply:
I would say, we should have at least one example for the Pythonic way and one for the YAML way for showcasing. Other than that, I fully agree with you.

Reply:
Agreed. In fact that Pythonic way should just be a rewrite of a simple YAMLic (coining a new word here) example so the user only learns one new thing and doesn't get distracted or confused